R/maraca.R

```diff
@@ -332,6 +398,19 @@ plot_maraca <- function(
       ),
       hjust = 0, vjust = 1.4, size = 3
     )
+
+    # Meta data on win odds will be added to plot
+    win_odds <- unname(win_odds)
+    params <- list(
+      "win_odds" = win_odds[1],
+      "lower_ci" = win_odds[2],
+      "upper_ci" = win_odds[3],
+      "p_value" = win_odds[4]
+    )
+
+    # Add win odds meta data as attribute so retrievable
+    attr(plot, "win_odds_parameters") <- params
```

Comment:
I generally don't like relying on attributes if it can be prevented. Is ggplot closed so you can't add more $properties? If it's closed, fair enough.

Reply:
Alternatively, you can create a S3 class from the ggplot object, adding the win_odds_parameters to that class. This siplifies also the validate method, because at that point you just use the S3 mechanism of resolution, and it simply won't be there for any other ggplot that is not a maracaPlot.

Reply:
Thanks, I opted to add the information by overwriting the label properties now. We wanted to keep the object as a ggplot object, both for the validate function and in order to be able to add further layers as a user.